arm-sql/compositeSql.json

```diff
@@ -13,8 +13,6 @@
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/databaseSecurityAlertPolicies.json",
-    "./2015-05-01-preview/swagger/blobAuditingPolicies.json",
```

Comment:
So these APIs aren't yet live in production, so I want to remove them from the compositeSql.json file for now until we're ready to actually use the client they'll generate.  Is that okay?